package-scripts.js

```diff
@@ -40,7 +40,7 @@ module.exports = {
     },
     validate: {
       description: 'This runs several scripts to make sure things look good before committing or on clean install',
-      script: 'p-s -p lint,build,cover && p-s check-coverage',
+      script: 'dist/bin/p-s.js -p lint,build,cover && p-s check-coverage',
```

Comment:
This is part of the temporary change. I'll change it back when we get the fix released.